### ISSUE TITLE
Problem with crontab

### ISSUE THREAD

**Issue by benoitgoyet** [jeremyephron/simplegmail#25]
Hy,

I'm trying to use simpegmail in a scheduled script using crontab (with a raspberry)
When I try the script without cron everything is fine. I can send a mail without any problem.
When U try to schedule it (using crontab -e), the call : 
`gmail = Gmail() # will open a browser window to ask you to log in and authenticate
` give me an issue with an unexpected end of the script (I check it with one breakpoint before and after)

Is this problem already known? Do you have a solution?

Thanks a lot.

Regards

**Comment by jeremyephron** [comment]
Just ran a script using `simplegmail` on `crontab` and everything worked fine. There is no inherent problem with running simplegmail within a cronjob. We can still try to get to the bottom of your problem though. Unexpected end of script sounds like something is wrong with the script and not the `Gmail()` call (perhaps a missing parenthesis).

Can you run the cronjob (just change the time to whatever the next minute is) and copy and paste the output here?

**Comment by benoitgoyet** [comment]
Thanks for your answer.

Here is my code : 
'
print("BP1")
import pdfkit
import sys
print("BP2")
from simplegmail import Gmail
from simplegmail.query import construct_query
print("BP3")
gmail = Gmail() # will open a browser window to ask you to log in and authenticate
print("BP4")
#definition des constantes pour filtrer les messages à traiter
expediteur = "###@gmail.com" # !!! changed for the forum
sujet = "export vers pdf"
print("BP5")
params = {
  "to": expediteur,
  "sender": "####@gmail.com", # !!!!changed for the forum
  "subject": "Résultat d'extraction",
  "msg_html": "<h1>Woah, my first email!</h1><br />This is an HTML email.",
  "msg_plain": "Hi\nThis is a plain text email.",
  "signature": True  # use my account signature
  }
print("BP6")
message = gmail.send_message(**params)
print("BP7")
`

Everything is fine from BP1 to BP3. But nothing else is displayed after.
But the full script is ok without cron...

Regards

**Comment by jeremyephron** [comment]
What is the code for your cronjob? If you are writing the output to a file, make sure to also write stderr to the file, so that you can see the error message.

**Comment by jeremyephron** [comment]
I just ran your code on a cronjob, and it worked fine. My hypothesis right now is that you don't have "gmail_token.json" in the right place. Can you please locate the "gmail_token.json" file and then specify its path as a parameter of `Gmail()` like
```
gmail = Gmail(creds_file='/path/to/gmail_token.json')
```

**Comment by benoitgoyet** [comment]
It’s strange, because when I use your proposition there is a mistake : TypeError: __init__() got an unexpected keyword argument ‘creds_file'

**Comment by jeremyephron** [comment]
@benoitgoyet Make sure to upgrade to the latest version 3.1.3. Run
```
python3 -m pip install --upgrade simplegmail
```

**Comment by jamesmalin** [comment]
I run simplegmail in a crontab on several servers as well and have no issues.

**Comment by jeremyephron** [comment]
Closing because I think the issue is resolved. @benoitgoyet feel free to reopen if you are still having issues.

**Comment by jeremyephron** [comment]
@benoitgoyet Something happened where the `creds_file` argument change was reverted (I don't know how this happened), but I've just fixed it in v3.1.5. Let me know if you were/are able to get things working
